src/util/isAbsolute.ts

```diff
@@ -0,0 +1,13 @@
+import { ABSOLUTE_TOKEN } from '../constants'
+import { Child, ThoughtContext } from '../types'
+
+/** Returns true if the thoughts or path is the absolute root. */
```

Comment:
Change to "Returns true if the thoughts or path is the absolute context"

The absolute context is not *inherently* a root, so "the absolute root" is a bit misleading; the current view has a root context which theoretically can point to any context (though I'm aware the current implementation isn't quite flexible enough for that).